src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {

```

Comment:
Don't begin class or method implementation with an empty line. Remove the redundant empty line at the beginning of the class definition.